_data/finance.yml

```diff
@@ -53,6 +53,13 @@ websites:
       img: ditech.png
       tfa: No
 
+    - name: Exact Online
+      url: https://start.exactonline.nl
```

Comment:
One last thing, for the url we list the main page of a site. That page can't be a login page and so I suppose the url value would be https://www.exact.com/

Reply:
Still, we don't list login pages in the `url` field so either make start.exactonline.nl redirect to something else than a login page or change the `url` value to https://www.exact.com/.